source/2019-12-13-ember-3-15-released.md

```diff
@@ -0,0 +1,131 @@
+
+Classic Ember components support using the `isVisible` property to toggle the
+visibility of a component's root element. This API was very common in early
+Ember 1.x applications but is rare today. In practice use of template bindings
+has supplanted use of the `isVisible` API.
+
+`isVisible` is deprecated as of Ember 3.15. 
+
+To transition away from this, we recommend using conditionals in a template:
+
+```handlebars
+{{#if showComponent}}
+  {{component}}
+{{/if}}
+
+{{! or }}
+<div hidden={{isHidden}}></div>
+
+```
+
+See the [deprecation guide for
+`isVisible`](https://deprecations.emberjs.com/v3.x#toc_ember-component-is-visible)
+for more information.
+
+### Deprecate `{{partial}}` (2 of 3)
+
+As of the introduction of [template-only
+components](https://api.emberjs.com/ember/3.15/classes/Component)
+and [Glimmer
+components](https://api.emberjs.com/ember/3.15/modules/@glimmer%2Fcomponent),
+as well as of the
+[template-only-glimmer-components](https://guides.emberjs.com/release/configuring-ember/optional-features/#toc_template-only-glimmer-components)
+optional feature the `{{partial}}` API in Ember no longer provides any unique
+benefits. Instead it has several downside including difficult to teach scoping 
+rules and poor performance.
+
+See the [deprecation guide for
+`{{partial}}`](https://deprecations.emberjs.com/v3.x#toc_ember-partial) for
+an example of how to migration existing use to component APIs.
+
+### Deprecate intimate `window.ENV` API (3 of 3)
+
+Ember applications long ago shifted to using `window.EmberENV` as the home of
+various configuration options. As of 3.15 application code which uses `window.ENV`
+for Ember configuration values will cause a deprecation warning to be issued.
+Support for this intimate API will be removed in 3.17.0.
+
+See [emberjs/ember.js#18441](https://github.com/emberjs/ember.js/pull/18441)
+for more details about this deprecation.
+
+For more details on changes in Ember.js 3.15, please review the [Ember.js 3.15.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.15.0).
+
+## Ember Data
+
+There were two new features and no deprecations for Ember Data v.3.15.0.
+
+### IDENTIFIERS turned on (1 of 2)
+
+(see https://github.com/emberjs/data/pull/6366) 
+
+### Full `links` object support for relationships (2 of 2)
+
+Features from [RFC]() were implemented that ensure full `links` object support for relationships, including: 
+- Preserves all existing behaviors attached to the related link
+- Adds `reference.links()` for accessing links
+- Adds `HasMany.links` and `PromiseHasMany.links` for accessing links
+- Ensures that links are returned from the cache in whichever format the user supplied (i.e., `{ href }` or string).
+
+For additional details on changes in Ember Data 3.15.0, please review the full
+[Ember Data 3.15.0 change log](https://github.com/emberjs/data/blob/release/CHANGELOG.md#release-3150-december-18-2019).
+
+## Ember CLI
```

Comment:
the new blueprint also dropped the Ember 3.4 ember-try scenario, but dropping support for existing addons should be considered a breaking change. we should highlight that existing addons should either keep the scenario, or do a major version bump if they drop support for older Ember versions.

see https://github.com/ember-cli/ember-cli/pull/8946